src/components/Footer/footerData.ts

```diff
@@ -0,0 +1,144 @@
+import { NavItemsFooter } from './FooterNav';
+
+export const navItems: NavItemsFooter[] = [
+  {
+    title: 'Group 1',
+    navItems: [
+      {
+        text: 'Item 1',
```

Comment:
we're using 'page group #' and 'page #' in other dummy component data. should we standardize?